build/Linux+BSD/portable/copy-libs

```diff
@@ -72,7 +74,7 @@ getCrossPlatformDependencies() {
   copyLib libQt5Sensors.so.5
   copyLib libQt5Sql.so.5
   copyLib libQt5Svg.so.5
-  copyLib libQt5WebChannel.so.5
```

Comment:
The webview was visible in the start centre? If so then you can leave out libQt5WebChannel.so.5, but it should still be in copy-libs, just commented out. You should also add a note to say why it was commented out. I.e. "Not present on Debian, and apparently not needed".